arch/um/os-Linux/process.c

```diff
@@ -223,9 +227,81 @@ void os_set_pdeathsig(void)
 
 #include <unistd.h>
 #include <sys/syscall.h>   /* For SYS_xxx definitions */
+extern unsigned long long physmem_size;
 
 int os_arch_prctl(int pid, int option, unsigned long *arg2)
 {
 	return syscall(SYS_arch_prctl, option, arg2);
 }
+
+int os_setup_seccomp(void)
+{
+	int err;
+	unsigned long __userspace_start = uml_reserved,
+		__userspace_end = uml_reserved +
+		physmem_size - (uml_reserved - uml_physmem);
```

Comment:
can be `high_physmem` (=__userspace_end = physmem_size - (uml_reserved - uml_physmem))